README.rst

```diff
@@ -1,8 +1,13 @@
 .. image::  https://raw.githubusercontent.com/jgorset/django-kronos/master/docs/banner.png
 
-.. image:: https://api.travis-ci.org/jgorset/django-kronos.svg?branch=master
+.. image:: https://coveralls.io/repos/github/joshblum/django-kronos/badge.svg?branch=master
+    :target: https://coveralls.io/github/joshblum/django-kronos?branch=master
```

Comment:
It appears this is necessary for the tests to pass since Travis looks for the correct coveralls repo!